app/src/main/java/org/oppia/app/walkthrough/WalkthroughFragmentListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.walkthrough
+
+/** Listener for when an activity should change a fragment position. */
+interface WalkthroughFragmentListener {
```

Comment:
For this & other new interfaces: please name them based on the action, e.g. ``WalkthroughPageChanger`` or similar.

This also isn't a listener in the traditional sense: it's being used to abstract functionality rather than registering for a callback.

Reply:
Okay!